deltachat-ios/Controller/Settings/AdvancedViewController.swift

```diff
@@ -462,6 +462,7 @@ internal final class AdvancedViewController: UITableViewController {
     private func updateCells() {
         showEmailsCell.detailTextLabel?.text = EmailOptionsViewController.getValString(val: dcContext.showEmails)
         videoChatInstanceCell.detailTextLabel?.text = VideoChatInstanceViewController.getValString(val: dcContext.getConfig("webrtc_instance") ?? "")
+        proxySettingsCell.detailTextLabel?.text = dcContext.isProxyEnabled ? String.localized("on") : nil
```

Comment:
There is probably a reason for showing a generic "On" instead of the used proxy?

Reply:
it is good enough, just a hint that jt is on.

the convretely used proxy is anyways mostly a big number zhat gets truncated